### PR TITLE
allegro.pl new smart logo selector in results

### PR DESCRIPTION
**Pull request by FadeMind**
https://allegro.pl/kategoria/akcesoria-laptop-pc-akcesoria-505?string=usb%20czytnik%20linii%20papilarnych

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -828,6 +828,7 @@ div.sob6jg.so1d89.mpof_ki
 .so15i8
 a[data-analytics-click-value="SellerAllListing"]
 .mpof_ki.m389_6m > h6
+i[class="_1e32a_sudoG"]
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:831]:
```suggestion
i._1e32a_sudoG
```

**Review by Gusted — APPROVED** [review]

**Comment by FadeMind** [comment]
@Gusted sorry, but `i._1e32a_sudoG` Does not work. 
`i[class="_1e32a_sudoG"]` work. 

Do another PR or you will fix self?

**Comment by Gusted** [comment]
> but `i._1e32a_sudoG` Does not work.
> `i[class="_1e32a_sudoG"]` work.

According to the provided screenshot `i[class="_1e32a_sudoG"]` shouldn't match, as the element has a trailing whitespace. While the existing applied selectors `._1e32a_sudoG` should work(with adding `i` as element).

**Comment by FadeMind** [comment]
@Gusted `i._1e32a_sudoG`  does not work.

**Comment by Gusted** [comment]
It should work, given the provided screenshot(the element doesn't show up locally for me). 

Either the screenshot is incorrect, or you're facing another bug.

**Comment by FadeMind** [comment]
@Gusted

**Comment by Gusted** [comment]
What's the result of `document.querySelectorAll('i[class="_1e32a_sudoG"]')` in the console?

Now I'm curious what I'm missing.

**Comment by FadeMind** [comment]
@Gusted where exactly put this code?

**Comment by Gusted** [comment]
ctrl + shift + i -> "Console" tab.

**Comment by FadeMind** [comment]
Typying in Console Chrome does nothing. 
Typing in style filter do:

**Comment by Gusted** [comment]
Ah, localization sorry. I meant please put the code in the "Konsola" tab.
image

**Comment by FadeMind** [comment]
@Gusted  I applied `i[class="_1e32a_sudoG"]` in DarkReader.

What do you need?

**Comment by Gusted** [comment]
> What do you need?

Yes, so what that tells us that there is no element that matches that selector. Could you try removing that selector from Dark Reader? it should yield the same results.

**Comment by FadeMind** [comment]
@Gusted I don't know W.T.F. here. 
I deleted `i[class="_1e32a_sudoG"]` from DR.
Now SMART! logo is inverted just like that (but when I create some day ago this PR -  was not inverted)

IMHO - please rollback this PR - weird.

**Comment by Gusted** [comment]
No worries, browsers and Dark Reader can be funky sometimes 😅.

**Pull request merged**